generators/typegen/generator.go

```diff
@@ -126,15 +158,57 @@ func (g *Generator) generateTypesForPackage(s *schema.Schema, genConfig *config.
 			}
 
 			enumsForGen = append(enumsForGen, xxx)
+		case schema.KindScalar:
+			log.Warnf("scalar type: %+v", t)
+
+			// Default scalars to int
+			createAs := "int"
+			skipTypeCreate := false
+
+			for _, p := range pkgConfig.Types {
+				if p.Name == t.GetName() {
```

Comment:
Should `t.GetName()` be cached in a local var vs looping through the function call?

Reply:
Update to cache the name outside the loop.